understanding/20/error-prevention-legal-financial-data.html

```diff
@@ -8,6 +8,15 @@
 <body>
    <h1>Understanding Error Prevention (Legal, Financial, Data)</h1>
    
+   <section id="brief">
+		<h2>In brief</h2>
+		<dl>
+		<dt>Objective</dt><dd>Users avoid submitting incorrect important information</dd>
+		<dt>Author task</dt><dd>Provide ways for users to confirm, correct or reverse data submissions</dd>
+		<dt>Key beneficiaries</dt><dd>Users with reduced sight, hearing, cognition, or motor function</dd>
```

Comment:
@gundulaniemann wrote
>I do not see the impact on users with reduced hearing.
Can you explain?

 All material for key beneficiaries is drawn from the User Benefits section, which here says "helps users with all disabilities"  
 This is the stock phrase I have used for "all disabilities". I've pondered just saying "All users" or "All users with disabilities". As mentioned before, de-prioritizing this work since I anticipate the Beneficiaries material being dropped.

Reply:
I see. "All users" is definitely more.